src/call_function.jl

```diff
@@ -1,53 +1,68 @@
-export @pm, call_function, call_method
+export @pm
+
+function CxxWrap.StdVector{CxxWrap.StdString}(vec::AbstractVector{<:AbstractString})
+    return CxxWrap.StdVector(convert.(CxxWrap.StdString, vec))
+end
 
 """
-    call_function(app::Symbol, func::Symbol, args...; void=false, kwargs...)
+    call_function([::Type{PropertyValue},] app::Symbol, func::Symbol, args...;
+        template_parameters=String[], unwrap=true, kwargs...)
```

Comment:
```suggestion
        template_parameters=String[], kwargs...)
```
remove unwrap argument here as well